src/main/cms/cms_menu_blackbox.c

```diff
@@ -122,7 +122,7 @@ static void cmsx_Blackbox_GetDeviceStatus(void)
     case BLACKBOX_DEVICE_FLASH:
         unit = "KB";
 
-        storageDeviceIsWorking = flashfsIsReady();
+        storageDeviceIsWorking = flashfsIsSupported();
```

Comment:
Sure it makes sense to make this switch here? `flashfsIsReady()` seems to be a stronger predicate than `flashfsIsSupported()`. Or probably use `flashfsIsSupported() && flashfsIsReady()`?

Reply:
I think there's no point checking for `flashfsIsReady` here, as flash should be idle (and ready) during CMS session if it exists.